packages/react-storage/src/components/StorageBrowser/components/PaginationButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { CLASS_BASE } from '../views/constants';
+import {
+  ButtonElement,
+  IconElement,
+  InputElementProps,
+} from '../context/elements';
+
+type ButtonType = 'previous' | 'next';
+
+interface PaginationButtonProps extends InputElementProps {
+  disabled: boolean;
+  onClick: () => void;
+  type: ButtonType;
+}
```

Comment:
Can you please make these changes and add the export key word please?
```suggestion
export interface PaginationButtonProps extends InputElementProps {
  isDisabled?: boolean;
  onClick?: () => void;
  type?: ButtonType;
}

Reply:
Made these changes. With the button variant and included icon being defined by `type`, I decided to return null if `type` isn't provided